### PR TITLE
Changes to fix issue with React virtualized WindowScroller by assigning the scrollElement property to a specific id

### PR DESCRIPTION
**Pull request by sg00dwin**
Events and list pages height is now corrected.

### CODE REVIEW

File: frontend/public/components/_project-overview.scss [hunk 1]
```diff
@@ -64,6 +64,7 @@
     }
 
     &::after {
+      align-self: center;
       content: '\f054';
```
Comment (rhamilto) [inline frontend/public/components/_project-overview.scss]: Since you've already got an unrelated change in here... This should make use of [the var](https://github.com/FortAwesome/Font-Awesome/blob/v4.7.0/scss/_variables.scss#L164) instead of directly using hex.

File: frontend/public/components/events.jsx [hunk 2]
```diff
@@ -415,7 +415,7 @@ class EventStream extends SafetyFirst {
           </div>
         </div>
         { count > 0 &&
-            <WindowScroller>
+            <WindowScroller scrollElement={document.getElementById('content-scrollable')}>
```
Comment (rhamilto) [inline frontend/public/components/events.jsx]: Wonder if there's a simple way to make `document.getElementById('content-scrollable')` a global?

File: frontend/public/components/app.jsx [hunk 3]
```diff
@@ -128,7 +128,7 @@ class App extends React.PureComponent {
       <div id="content">
         <Route path={namespacedRoutes} component={NamespaceSelector} />
         <GlobalNotifications />
-        <div className="content-scrollable">
+        <div id="content-scrollable">
```
Comment (spadgett) [inline frontend/public/components/app.jsx]: Is it possible to only add the scrollable div in the overview instead of every page?

Comment (sg00dwin) [reply frontend/public/components/app.jsx]: It is possible but that will introduce inconsistent scrolling behavior regarding the project selector. The `content-scrollable` doesn't contain the project selector, meaning on the overview it will always be visible and the rest of the remaining pages it will be within the scrollable area.

File: frontend/public/components/events.jsx (hunk 2 above)
Comment (spadgett) [reply frontend/public/components/events.jsx]: I'm hesitant to hold on to the actual element reference. Not sure if it ever could be removed from the DOM. It seems safer to use `getElementById` when we need it.

File: frontend/public/components/app.jsx (hunk 3 above)
Comment (sg00dwin) [reply frontend/public/components/app.jsx]: I found one and currently the modal body doesn't have scroll enabled, so that's a bug.

modal-from-overview

When scroll is enabled I'm not see the "background" panels scroll effected by the modal.
chrome-scroll-chaining-overview-sidenav

Comment (spadgett) [reply frontend/public/components/app.jsx]: I don't know if this was a problem before, but I can scroll content behind dialogs on my phone. For instance, open the "Create Project" dialog. You can scroll the page behind the dialog content on iOS.

Comment (spadgett) [reply frontend/public/components/app.jsx]: OK I was able to reproduce on 3.11 as well, so that's not new. We should open a bug to track.

**Comment by spadgett** [comment]
jenkins rebuild

File: frontend/public/components/app.jsx (hunk 3 above)
Comment (spadgett) [reply frontend/public/components/app.jsx]: Looks like I already opened a but for this, CONSOLE-658.

File: frontend/public/components/_project-overview.scss (hunk 1 above)
Comment (sg00dwin) [reply frontend/public/components/_project-overview.scss]: Pulled into separate pr 
https://github.com/openshift/console/pull/608

**Comment by spadgett** [comment]
Pushed a workaround to suppress the height warning
